other/nightly/install.sh

```
@@ -438,24 +438,54 @@ fi
         fi
     fi
+    set -e
+}
+
+install_docker_manually() {
+    case "$OS_TYPE" in
+    "ubuntu" | "debian" | "raspbian")
+        apt-get update
+        apt-get install -y ca-certificates curl
+        install -m 0755 -d /etc/apt/keyrings
+        curl -fsSL https://download.docker.com/linux/$OS_TYPE/gpg -o /etc/apt/keyrings/docker.asc
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)